### DISCUSSION TITLE
2.2.14升级后，json schema中array类型的设置default字段无法默认展示

### DISCUSSION THREAD

**Discussion by hetianyu0315** [alibaba/formily#3615] [Q&A 2.x] [answered]
代码示例：
[![Edit array-default-value-reset-issue (forked)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/array-default-value-reset-issue-forked-m8fsne?fontsize=14&hidenavigation=1&theme=light)

问题复现：
1、当formily资源切换到2.2.13时，重新载入显示正常。
![image](https://user-images.githubusercontent.com/69338208/206636481-6193edd5-578f-4283-a535-e82cebee5731.png)
2、当formilyformily资源切换到2.2.14时，重新载入后表单无法正常渲染默认值
![image](https://user-images.githubusercontent.com/69338208/206636784-7833f40e-da30-422b-b18f-4c07a459e694.png)

**Comment by TinaZbh** [comment]
解决了吗？我也遇到了类似的问题

**Comment by hetianyu0315** [comment]
还没尝试，我强行降级到2.2.13处理的





来自钉钉专属商务邮箱------------------------------------------------------------------
***@***.***>
日　期：2022年12月27日 18:06:45
***@***.***>
***@***.***>; ***@***.***>
主　题：Re: [alibaba/formily] 2.2.14升级后，json schema中array类型的设置default字段无法默认展示 (Discussion #3615)

解决了吗？我也遇到了类似的问题

**Comment by janryWang** [comment, marked as answer]
已经修复

**Comment by hetianyu0315** [comment]
收到，感谢





来自钉钉专属商务邮箱------------------------------------------------------------------
***@***.***>
日　期：2023年01月03日 23:58:31
***@***.***>
***@***.***>; ***@***.***>
主　题：Re: [alibaba/formily] 2.2.14升级后，json schema中array类型的设置default字段无法默认展示 (Discussion #3615)

已经修复
